packages/ui-options/src/Options/Item/props.ts

```diff
@@ -146,5 +146,5 @@ const allowedProps: AllowedPropKeys = [
   'children'
 ]
 
-export type { OptionsItemProps, OptionsItemStyle }
+export type { OptionsItemProps, OptionsItemStyle, ItemProps }
```

Comment:
Please prefix all variable names that are exported from the package with the component's name. Since `OptionsItemProps` is already taken, I would call this `OptionsItemMainProps` or sth like this.

Reply:
fixed